src/Tiqr/Legacy/TiqrService.php

```diff
@@ -35,36 +39,22 @@
  * Wrapper around the legacy Tiqr service.
  *
  * @SuppressWarnings(PHPMD.TooManyPublicMethods)
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
  * It's Legacy.
  */
 final class TiqrService implements TiqrServiceInterface
 {
-    const ENROLL_KEYS_SESSION_NAME = 'enrollment-session-keys';
-    /**
-     * @var \Tiqr_Service
-     */
-    private $tiqrService;
-    private $tiqrStateStorage;
-    private $session;
-    private $logger;
-
-    /**
-     * @var string
-     */
-    private $accountName;
+    public const ENROLL_KEYS_SESSION_NAME = 'enrollment-session-keys';
+    private SessionInterface $session;
```

Comment:
Since whe have a request stack, maybe SessionInterface is obsolete?

Reply:
It is not; 

See line 426 and onwards. We init the session field from the request stack.